### ISSUE TITLE
Too much help

### ISSUE THREAD

**Issue by dertuxmalwieder** [fcorbelli/zpaqfranz#24]
If I accidentally set the switches and the other parameters in the wrong order, `zpaqfranz` tells me *three times* what the correct order is:

```
> ..\..\Downloads\zpaqfranz.exe a -m5 upload.zpaq .\upload
zpaqfranz v55.6-experimental (HW BLAKE3), SFX64 v55.1, compiled Jul 26 2022
Usage: zpaqfranz command archive[.zpaq] files|directory... -switches...
             h: **** Help on Help ****|  With great power comes great  ...  help!
             a: Append files          |          t: Test (integrity)
             x: Extract versions      |          l: List files
             v: Verify on filesystem  |          i: Info (show versions)
           sfx: Create SFX (Windows)  |         rd: Remove  'highlander'  folders
 c d0 d1 d2...: Compare d0 to d1,d2.. | s d0 d1 d2: Cumulative size of d0, d1, d2
 r d0 d1 d2...: Mirror  d0 in d1...   |       d d0: Deduplicate d0  WITHOUT MERCY
 z d0 d1 d2...: Delete empty dirs     |        m X: Merge multipart archive
          f d0: Fill /wipe free space |     utf d0: Detox filenames  in  d0
  sum d0 d1...: Hashing/deduplication |     dir d0: Win dir (/s /a /os /od)
     n d0 -n X: Keep X files in d0    |          b: CPU benchmarking (-all)
   rsync d0 d1: Purge temporary rsync | cp X -to Y: Copy files (w/wildcards) to Y
          trim: Trim incomplete add() |    dirsize: Get size of archive's folders
      password: Add/remove/change pwd |   q z:\foo: VSS-archive C: in z:\foo.zpaq
                                 Main | switches
      -all [N]: All versions N digit  |     -key X: Use/set archive password to X
 -mN -method N: 0..5= faster..better  |     -force: Always overwrite (extraction)
         -test: Test (extract/add)    |      -kill: Allow destructive (NO dryrun)
    -to out...: Prefix files to out   |   -until N: Roll back to N'th version
Usage: zpaqfranz command archive[.zpaq] files|directory... -switches...
             h: **** Help on Help ****|  With great power comes great  ...  help!
             a: Append files          |          t: Test (integrity)
             x: Extract versions      |          l: List files
             v: Verify on filesystem  |          i: Info (show versions)
           sfx: Create SFX (Windows)  |         rd: Remove  'highlander'  folders
 c d0 d1 d2...: Compare d0 to d1,d2.. | s d0 d1 d2: Cumulative size of d0, d1, d2
 r d0 d1 d2...: Mirror  d0 in d1...   |       d d0: Deduplicate d0  WITHOUT MERCY
 z d0 d1 d2...: Delete empty dirs     |        m X: Merge multipart archive
          f d0: Fill /wipe free space |     utf d0: Detox filenames  in  d0
  sum d0 d1...: Hashing/deduplication |     dir d0: Win dir (/s /a /os /od)
     n d0 -n X: Keep X files in d0    |          b: CPU benchmarking (-all)
   rsync d0 d1: Purge temporary rsync | cp X -to Y: Copy files (w/wildcards) to Y
          trim: Trim incomplete add() |    dirsize: Get size of archive's folders
      password: Add/remove/change pwd |   q z:\foo: VSS-archive C: in z:\foo.zpaq
                                 Main | switches
      -all [N]: All versions N digit  |     -key X: Use/set archive password to X
 -mN -method N: 0..5= faster..better  |     -force: Always overwrite (extraction)
         -test: Test (extract/add)    |      -kill: Allow destructive (NO dryrun)
    -to out...: Prefix files to out   |   -until N: Roll back to N'th version
Usage: zpaqfranz command archive[.zpaq] files|directory... -switches...
             h: **** Help on Help ****|  With great power comes great  ...  help!
             a: Append files          |          t: Test (integrity)
             x: Extract versions      |          l: List files
             v: Verify on filesystem  |          i: Info (show versions)
           sfx: Create SFX (Windows)  |         rd: Remove  'highlander'  folders
 c d0 d1 d2...: Compare d0 to d1,d2.. | s d0 d1 d2: Cumulative size of d0, d1, d2
 r d0 d1 d2...: Mirror  d0 in d1...   |       d d0: Deduplicate d0  WITHOUT MERCY
 z d0 d1 d2...: Delete empty dirs     |        m X: Merge multipart archive
          f d0: Fill /wipe free space |     utf d0: Detox filenames  in  d0
  sum d0 d1...: Hashing/deduplication |     dir d0: Win dir (/s /a /os /od)
     n d0 -n X: Keep X files in d0    |          b: CPU benchmarking (-all)
   rsync d0 d1: Purge temporary rsync | cp X -to Y: Copy files (w/wildcards) to Y
          trim: Trim incomplete add() |    dirsize: Get size of archive's folders
      password: Add/remove/change pwd |   q z:\foo: VSS-archive C: in z:\foo.zpaq
                                 Main | switches
      -all [N]: All versions N digit  |     -key X: Use/set archive password to X
 -mN -method N: 0..5= faster..better  |     -force: Always overwrite (extraction)
-- More (q, Q or control C to exit) --
```

I think one time would be enough?

**Comment by fcorbelli** [comment]
repetita iuvant!

**Comment by fcorbelli** [comment]
Fixed in 55.8b (to be released)
